docs/core/option.md

```diff
@@ -111,4 +203,40 @@ IReadOnlyList<int> list = [2, 4, 6];
 Option<IReadOnlyList<int>> listOfOnlyEvenNumbers =
     list.Traverse(x => x % 2 == 0 ? Some(x) : None);
 // Some([2, 4, 6])
+```
+
+### Sequence
+
+Use `Sequence` to traverse without the mapping step.
+
+```csharp
+IReadOnlyList<Option<int>> list = [Some(2), Some(4), Some(6)];
+
+Option<IReadOnlyList<int>> listOfOnlyEvenNumbers =
+    list.Sequence();
+// Some([2, 4, 6])
+```
+
+## Prelude
+
+The `Prelude` class provides the following functions for `Option`:
+
+### Some / None
+
+Returns a wrapped value or an empty `Option`.
+
+```csharp
+public Option<string> OnlyNonWhitespace(string? value) =>
+    string.IsNullOrWhiteSpace(value)
+        ? None
+        : Some(value);
+```
+
+### Optional
+
+Use `Optional` to convert a nullable value to `Option`. Null value will result to an empty option.
```

Comment:
```suggestion
Use `Optional` to convert a nullable value to `Option`. If the input value is **not** `null`, the result will be `Some(value)`, otherwise the result will be an empty option (`None`).
```

Reply:
Done